mms2/README.md

```diff
@@ -0,0 +1,43 @@
+## Scylla in Docker for the Mutant Monitoring Blog Series
+
+### Instructions for setting up a Scylla Cluster from this repo
+
+```
+cd mms2
+docker-compose up -d
+```
+
+Run bash in the node:
+```
+docker exec -it mms_scylla-node1_1 bash
+```
+
+Followed by scylla commands, like
+```
+> nodetool status
+```
+or
+```
+> cqlsh
+```
+
+### Destroying the Scylla Cluster 
+```
+cd mms2
+docker-compose kill
+docker-compose rm -f
+```
+### Importing the MMS keyspaces and data automatically 
+
+```
+docker exec scylla-node1 cqlsh -f /mutant-data.txt
+```
+
+Then re-run docker-compose:
```

Comment:
Why do you need to re-run docker to load data?

Reply:
I updated the readme file accordingly